coalib/output/Interactions.py

```diff
@@ -28,3 +45,35 @@ def fail_acquire_settings(log_printer, settings_names_dict):
 
         logging.error(msg)
         raise AssertionError(msg)
+
+
+def highlight_text(no_color, text, style, lexer=TextLexer()):
```

Comment:
so now instead of `format_lines` being duplicated, `coalib/output/ConsoleInteraction.py:highlight_text` is being duplicated here.

How is this better?

Reply:
Sorry, I've fixed it now.
No duplicates of `highlight_text()` anymore :)

Reply:
So, why the Interactions module?